test/fast/FastFrontendFacet.test.ts

```diff
@@ -5,9 +5,9 @@ import { BigNumber } from 'ethers';
 import { deployments, ethers } from 'hardhat';
 import { FakeContract, smock } from '@defi-wonderland/smock';
 import { SignerWithAddress } from 'hardhat-deploy-ethers/signers';
-import { tenThousand } from '../utils';
+import { zero, tenThousand } from '../utils';
```

Comment:
I think at some point we'll need some helper function that accepts a `string | number | bignumber` so we don't need all these variables around. Initially I thought it would make tests more readable.